specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdnwebapplicationfirewall.json

```diff
@@ -599,7 +602,7 @@
           "type": "string",
           "enum": [
             "RemoteAddr",
-            "Country",
+            "SocketAddr",
```

Comment:
Is this the change for preview version?

Reply:
This change is for both the preview and stable versions. Removing `Country` is not a breaking change because `Country` was originally added by mistake and was never supported by our service. `SocketAddr` is needed for our public preview, so we're including it in the preview version as well as stable.